.drone.yml

```diff
@@ -5,11 +5,14 @@ bootstrap_common: &bootstrap_common
     - mkdir -p /opt/metwork-mfcom-${DRONE_BRANCH}
     - ./bootstrap.sh /opt/metwork-mfcom-${DRONE_BRANCH} /opt/metwork-mfext-$${MFEXT_BRANCH}
     - cat adm/root.mk
+    - CACHE_HASH=`/opt/metwork-mfext-${DRONE_BRANCH}/bin/_build_cache_hash.sh .`
```

Comment:
We have to use $${MFEXT_BRANCH} and not ${DRONE_BRANCH} here because they can be different in some cases

Reply:
Done